### PR TITLE
Scour 0.32

### PR DESCRIPTION
**Pull request by Eitot**
Add formula for Scour, per #47155.

### CODE REVIEW

File: Library/Formula/scour.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/scour", "--help"
```
Comment (DomT4) [inline Library/Formula/scour.rb]: Can we improve this test at all? We have a test_fixture for an `.svg` file so perhaps copy that to the testpath and then "scrub" it or something?

Comment (Eitot) [inline Library/Formula/scour.rb]: Can you explain? Then I’ll have a look. The problem is of course that the file needs to be written somewhere as well. Is that something this test function should do?

Comment (dunn) [inline Library/Formula/scour.rb]: It can write it to the temporary `testpath`, since that'll be removed.

Comment (DomT4) [inline Library/Formula/scour.rb]: Fairly basic example [here](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/pdf2svg.rb#L25).

Comment (DomT4) [inline Library/Formula/scour.rb]: Er, [here](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/svg2png.rb#L26-L27) even :sweat:.

Comment (Eitot) [inline Library/Formula/scour.rb]: Thanks, done!

**Comment by apjanke** [comment]
Merged. Thank you for your contribution to Homebrew!

**Pull request closed without merging**